src/test/optimizer/strategy/between_composition_rule_test.cpp

```diff
@@ -0,0 +1,258 @@
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "expression/expression_functional.hpp"
+#include "logical_query_plan/join_node.hpp"
+#include "logical_query_plan/lqp_translator.hpp"
+#include "logical_query_plan/predicate_node.hpp"
+#include "logical_query_plan/projection_node.hpp"
+#include "logical_query_plan/sort_node.hpp"
+#include "logical_query_plan/stored_table_node.hpp"
+#include "logical_query_plan/union_node.hpp"
+#include "operators/get_table.hpp"
+#include "optimizer/strategy/between_composition_rule.hpp"
+#include "optimizer/strategy/strategy_base_test.hpp"
+#include "statistics/column_statistics.hpp"
+#include "statistics/table_statistics.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "storage/storage_manager.hpp"
+
+#include "utils/assert.hpp"
+
+#include "logical_query_plan/mock_node.hpp"
+
+using namespace opossum::expression_functional;  // NOLINT
+
+namespace opossum {
+
+class BetweenCompositionTest : public StrategyBaseTest {
+ protected:
+  void SetUp() override {
+    const auto table = load_table("resources/test_data/tbl/int_int_int.tbl");
+    StorageManager::get().add_table("a", table);
+    _rule = std::make_shared<BetweenCompositionRule>();
+    node = StoredTableNode::make("a");
+
+    a = LQPColumnReference{node, ColumnID{0}};
+    b = LQPColumnReference{node, ColumnID{1}};
+    c = LQPColumnReference{node, ColumnID{2}};
+  }
+
+  std::shared_ptr<StoredTableNode> node;
+  LQPColumnReference a, b, c;
```

Comment:
In l. 46 and 47 an "_" is probably missing before the variable names. Also, according to the [style guide](https://github.com/hyrise/hyrise/blob/master/CONTRIBUTING.md), the names could be more descriptive.